emails/2023-06-30.md

```diff
@@ -0,0 +1,25 @@
+This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 
+
+It's a long weekend here in the USA so happy 4th to those who celebrate. I begrudgingly gave the team Monday off as well so expect slow response until next Wednesday. This is how I labeled the extra day off in our HR software.
+
+![Extra Day Off](../images/extra-day-off.png)
+
+### Best Hospital Price Database Yet
+
+As our regular readers know, we've been working on hospital price transparency [since 2021](https://www.dolthub.com/repositories/dolthub/hospital-price-transparency). We've built a [number of databases](https://www.dolthub.com/profile/discover?q=transparency) learning more about the hospital pricing system as we go. We're currently building our [best, most accurate database](https://www.dolthub.com/repositories/dolthub/transparency-in-pricing) yet. This database has a refined schema, automated integration tests of Pull Requests, and an experienced reviewer ensuring quality. 
+
+This week we published [a blog on how to use it](https://www.dolthub.com/blog/2023-06-28-how-not-to-hpt/). Have at it data scientists! Find interesting stories!
+
+### Air Gapped Networks
+
+We've had a number of people poke around with Dolt as a solution for getting data in and out of air-gapped networks. After doing some research Git and Dolt seem like ideal tools for managing files and tables across an air-gapped boundary. So, I [wrote about the use case](https://www.dolthub.com/blog/2023-06-26-air-gap-network/). But really, it was just an excuse to use an image of the [Mt. Baker Road Gap](https://www.youtube.com/watch?v=XOsXU_gbGI8).
+
+[![Mt Baker Road Gap](../images/air-gap.jpg)](https://www.dolthub.com/blog/2023-06-26-air-gap-network/)
+
+### Faster Than MySQL?
+
+We were debugging a query for a customer and we found we were faster than MySQL for that query. Generally, we are about [2X slower than MySQL](https://docs.dolthub.com/sql-reference/benchmarks/latency) since Dolt 1.0. I asked [Max](https://www.dolthub.com/team#max), our SQL analyzer owner, to write [a blog about why this query was faster](https://www.dolthub.com/blog/2023-06-23-faster-than-mysql/). For good measure he threw in a Postgres comparison. If you want to dive deep into SQL engines, [this blog](https://www.dolthub.com/blog/2023-06-23-faster-than-mysql/) is for you.
```

Comment:
```suggestion
We were debugging a query for a customer and we found we were faster than MySQL for that query. Generally, we are about [2X slower than MySQL](https://docs.dolthub.com/sql-reference/benchmarks/latency) since Dolt 1.0. I asked [Max](https://www.dolthub.com/team#max), our SQL analyzer owner, to write [a blog about why this query was faster](https://www.dolthub.com/blog/2023-06-23-faster-than-mysql/). For good measure, he threw in a Postgres comparison. If you want to dive deep into SQL engines, [this blog](https://www.dolthub.com/blog/2023-06-23-faster-than-mysql/) is for you.
```